scripts/init.d-xapissl

```diff
@@ -111,30 +89,13 @@ debug = authpriv.5
 accept = ${ACCEPT}
 connect = 80
 cert = ${PEMFILE}
-ciphers = !SSLv2:${GOOD_CIPHERS}${EXTRA_CIPHERS}
+ciphers = ${GOOD_CIPHERS}
 curve = secp384r1
 TIMEOUTclose = 1
 options = CIPHER_SERVER_PREFERENCE
 options = NO_SSLv2
 EOF
-    if [ $BACK_COMPAT = 0 ]; then
-        writec 'options = NO_SSLv3'
-        # Ancient (pre 4.16) stunnel does not support sslVersion.
-        if [ ${ANCIENT_STUNNEL} = 0 ]; then
-            writec 'sslVersion = TLSv1.2'
-        fi
-    else
-        if [ ${ANCIENT_STUNNEL} = 0 ]; then
-            # stunnel does not let us specify "SSLv3 and TLSv1.2 only".
-            # With multiple sslVersion entries, only the last is used.
-            writec 'sslVersionMax = TLSv1.2'
-            writec 'sslVersionMin = TLSv1'
-
-            # If we had v5.06 or newer we would need to allow SSLv3 with
-            # writec 'options -NO_SSLv3'
-            # but all earlier versions choke on that option.
-        fi
-    fi
+    writec 'sslVersionMin = TLSv1.2'
```

Comment:
Add `sslVersionMax = TLSv1.2` too, otherwise you also enable TLSv1.3 and we're not ready for that one yet.